src/mixins/date-mixin.ts

```diff
@@ -10,8 +10,10 @@ import { CommonMixin } from '@/mixins'
 /**
  * Mixin that provides some useful date utilities.
  */
-@Component({})
-export default class DateMixin extends Mixins(CommonMixin) {
+@Component({
+  extends: CommonMixin
+})
+export default class DateMixin extends CommonMixin {
```

Comment:
I had to do this to work around some incompatibility with the mixin using another mixin.

Ref: https://github.com/apertureless/vue-chartjs/issues/495#issuecomment-489400832